### PR TITLE
WIP: Activate short button in taker UI

### PR DESCRIPTION
**Pull request by rishflab**
Screenshot from 2022-02-24 10-49-07

### CODE REVIEW

File: taker-frontend/src/components/Trade.tsx [hunk 1]
```diff
@@ -99,10 +99,13 @@ export default function Trade({
     }, [userHasEdited, minQuantity, setQuantity]);
 
     let [onLongSubmit, isLongSubmitting] = usePostRequest<CfdOrderRequestPayload>("/api/cfd/order");
+    // todo: point this at the /api/cfd/order/short
```
Comment (rishflab) [inline taker-frontend/src/components/Trade.tsx]: dont need this

File: taker-frontend/src/components/History.tsx [hunk 2]
```diff
@@ -62,6 +62,7 @@ interface CfdDetailsProps {
 }
 
 const CfdDetails = ({ cfd, connectedToMaker, displayCloseButton }: CfdDetailsProps) => {
```
Comment (klochowicz) [inline taker-frontend/src/components/History.tsx:64]: nit: this could be a separate commit (or at the very least it should be mentioned in the description) - it's a bit unexpected looking at the PR that you're adding the position to all the tiles.

File: taker-frontend/src/components/Trade.tsx [hunk 3]
```diff
@@ -156,6 +162,7 @@ export default function Trade({
     }
 
     const confirmRef = useRef<HTMLButtonElement | null>(null);
```
Comment (klochowicz) [inline taker-frontend/src/components/Trade.tsx]: nit: probably rename to `confirmRefLong` for symmetry

File: taker-frontend/src/components/Trade.tsx [hunk 4]
```diff
@@ -239,6 +243,19 @@ export default function Trade({
                                 >
                                     Long
                                 </Button>
+                                <Button
+                                    // canSubmitShort is not a problem
+                                    disabled={!canSubmitShort}
+                                    colorScheme="red"
+                                    size="lg"
+                                    // using onShortOpen fails
```
Comment (klochowicz) [inline taker-frontend/src/components/Trade.tsx]: what are these comments? are there TODOs?

File: taker-frontend/src/components/Trade.tsx [hunk 5]
```diff
@@ -323,6 +341,96 @@ export default function Trade({
                                         </ModalFooter>
                                     </ModalContent>
                                 </Modal>
+
+                                <Modal
```
Comment (klochowicz) [inline taker-frontend/src/components/Trade.tsx:342]: after getting it working, I guess the next step would be extracting this duplicated code and passing the parameters accordingly

**Review by klochowicz — COMMENTED** [review]
good start!

File: taker-frontend/src/components/Trade.tsx (hunk 4 above)
Comment (rishflab) [reply taker-frontend/src/components/Trade.tsx]: oh no they can be removed. i put them there to document an issue i was having which is now resolved
